adam-core/src/test/resources/sorted-variants.vcf

```diff
@@ -0,0 +1,74 @@
+##fileformat=VCFv4.2
```

Comment:
We already have a `adam-core/src/test/resources/sorted.vcf`. Why is this file needed?